templates/kernel/lifecycle.html

```diff
@@ -0,0 +1,784 @@
+        <pre><code>sudo apt-get install --install-recommends linux-generic-lts-xenial xserver-xorg-core-lts-xenial xserver-xorg-lts-xenial xserver-xorg-video-all-lts-xenial xserver-xorg-input-all-lts-xenial libwayland-egl1-mesa-lts-xenial</code></pre>
+        <p>Server:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-lts-xenial</code></pre>
+        <p>The 14.04.2 and newer point releases ship with an updated kernel and X stack by default. If you have installed with older media, you can use these instructions to install the newer HWE kernel derived from 16.04 (Xenial).</p>
+        <p>If you run a multiarch desktop (for example, i386 and amd64 on amd64, for gaming or Wine), you may find you need a slightly more involved command:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-lts-xenial xserver-xorg-core-lts-xenial xserver-xorg-lts-xenial xserver-xorg-video-all-lts-xenial xserver-xorg-input-all-lts-xenial libwayland-egl1-mesa-lts-xenial libgl1-mesa-glx-lts-xenial libgl1-mesa-glx-lts-xenial:i386 libglapi-mesa-lts-xenial:i386</code></pre>
+      </div>
+
+      <div class="p-tabs__content" id="installation-12-04" role="tabpanel" aria-labelledby="installation-12-04-tab">
+        <h2>Ubuntu 12.04 LTS &mdash; Precise Pangolin</h2>
+        <p>Desktop:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-lts-trusty xserver-xorg-lts-trusty libgl1-mesa-glx-lts-trusty</code></pre>
+        <p>Server:</p>
+        <pre><code>sudo apt-get install --install-recommends linux-generic-lts-trusty</code></pre>
+        <p>The 12.04.2 and newer point releases shipped with an updated kernel and X stack by default. The hardware enablement stack is comprised of the newer kernel and X stacks from 14.04 (Trusty).</p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip is-bordered">
+    <div class="row">
+      <div class="col-8">
+        <h2>Check your support status</h2>
+        <p>To determine if your install is still supported, use <code>hwe-support-status</code> as documented on the <a href="/about/release-cycle">Ubuntu release cycle</a>.</p>
+        <p>
+          <code>hwe-support-status --verbose</code>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--light is-bordered">
+    <div class="row">
+      <div class="col-8">
+        <h2>Kernel release schedule</h2>
+        <div class="u-hide--small" id="kernel-schedule"></div>
+        <table class="u-hide--medium u-hide--large">
+          <thead>
+            <tr>
+              <td>&nbsp;</td>
+              <th scope="col">Released</th>
+              <th scope="col">End of Life</th>
+              <th scope="col">Extended security maintenance</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><strong>Ubuntu 14.04 LTS (v3.13)</strong></td>
+              <td>Apr 2014</td>
+              <td>Apr 2019</td>
```

Comment:
Should match the columns in the head so add `<td>&nbsp;</td>` to the end here for each row.